### PR TITLE
fix: add info about component classes

### PR DESCRIPTION
**Pull request by AnnaRybkina**

### CODE REVIEW

**Review by siljec — APPROVED** [review]

File: docs/getting-started/developers/index.md [hunk 1]
```diff
@@ -48,23 +48,41 @@ When setting up Warp in your project, you can choose to create a `uno.config.js`
...
 By default, UnoCSS will automatically look in the root directory of your project for `uno.config.[js,ts,mjs,mts]` or `unocss.config.[js,ts,mjs,mts]`.
 
+
+#### Components
+
+If you are using Warp components there are two additional steps to do.
+
+1. You'll need to add `https://assets.finn.no/pkg/@warp-ds/css/v1/components.css` in the head of your app or in the template if you are using such. This stylesheet is needed to add styling to Warp components.
+
+2. If you are NOT using Warp classes directly an only use our components you can skip this step. If you want to optimize the size of the produced css and you are using Warp's classes and want to prevent duplicate classes to be created - an `externalizeClasses` option needs to be added to the UnoCSS config and add classes from compoenent-classes as `externalClasses`. See example below and [check here for more info](https://github.com/warp-ds/drive#externalizeclasses):
```
Comment (vsandvold) [inline docs/getting-started/developers/index.md:73]: I find this step a bit unclear and confusing. Is it possible to have both of these steps as recommended, removing the question about using classes directly or not?

"Do this and this" is much easier to understand than "Do this, and maybe, if you are not doing that, do this as well".

**Comment by vsandvold** [comment]
I think you're doing a great job documenting the Warp framework. There's a lot of options though, resulting in a lot of "if's and but's" in the getting started guide.

Given more time, would you consider identifying the most common use cases, describing more clearly the steps required? It could be more like listing the options at the beginning, instead of listing the options at each step.

File: docs/getting-started/developers/index.md (hunk 1 above)
Comment (AnnaRybkina) [reply docs/getting-started/developers/index.md:73]: I see your point and I agree. As I understand now we are leaning towards the option to always include components.css for every podlet. So Option 2 will, in that case, disappear as purging will always be done on our side.
I'll rephrase

Comment (AnnaRybkina) [reply docs/getting-started/developers/index.md:73]: But it is also so difficult for me to write something down without if's and but's when we have so many use cases. But I'll try

**Pull request closed without merging**